packages/lib-classifier/src/store/subjects/Subject/FreehandLineReductions/FreehandLineReductions.spec.js

```diff
@@ -0,0 +1,116 @@
+import { expect } from 'chai'
+import sinon from 'sinon'
+import { reducedEmptySubject, reducedSubject } from './mocks'
```

Comment:
This line is erroring. The mocks don't exist.

Reply:
Didn't fix tests as part of first POC commit. Fixed now.